### DISCUSSION TITLE
line lengths for code in examples-gallery

### DISCUSSION THREAD

**Discussion by Peter230655** [csu-hmc/opty#192] [Q&A]
Python recommends that lines of code be shorter than 80 characters, and line length of comments be shorter than 73 characters.
What about comments appearing under # %%? What is the maximum length for them? 
**Thanks!**

**Comment by moorepants** [comment]
We keep lines lengths less than 80 characters in almost all files in the opty source code. There are some exceptions.

**Comment by Peter230655** [comment]
**Thanks!** I even managed to set the ruler in VSC at 79!
